superset-frontend/plugins/plugin-chart-echarts/src/utils/helper.ts

```diff
@@ -0,0 +1,5 @@
+export const convertNumber = (value: string | number) => {
```

Comment:
This new file needs the Apache license comment block added to the top (just copy/paste from nearly any JS/TS file)

Reply:
Also a small nitpick, but maybe "convertInteger" would be a better name, in case we ever need to do the same for a float.